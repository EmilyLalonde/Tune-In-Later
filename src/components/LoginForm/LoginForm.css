.formInput {
  border: 3px solid #002236;
  border-radius: 6px;
  display: block;
  font-size: 20px;
  height: 50px;
  margin-bottom: 20px;
}

.login-form {
  display: grid;
  margin-left: 30%;
  margin-top: 5%;
  width: 40%;
}

.login-button {
  background-color: #36abdd;
  border: 3px solid #002236;
  border-radius: 6px;
  color: #002236;
  font-size: 30px;
  height: 50px;
}

