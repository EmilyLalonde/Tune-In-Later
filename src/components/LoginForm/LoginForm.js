import React, { Component } from 'react'
import './LoginForm.css';
import { loginUser } from '../../apiCalls/apiCalls';

class LoginForm extends Component {
  constructor() {
    super()
    this.state = {
      email: '',
      password: ''
    }
  }

  handleTyping = (e) => {
    this.setState({
      [e.target.name] : e.target.value
    })
  }

  handleSubmit = (e) => {
    e.preventDefault();
    const user = {
      email: this.state.email,
      password: this.state.password
    }

    loginUser(user)
  }

  render() {
    return (
      <form>
        <input
          type='text'
          name='email'
          placeholder='Please enter email address'
          value={this.state.email}
          onChange={this.handleTyping}
          className='formInput'
        />
        <input 
          type='text'
          name='password'
          placeholder='Please enter password'
          value={this.state.password}
          onChange={this.handleTyping}
          className='formInput'
        />
        <button
          onClick={this.handleSubmit}
        >SUBMIT</button>
      </form>
    )
  }
}

export default LoginForm;