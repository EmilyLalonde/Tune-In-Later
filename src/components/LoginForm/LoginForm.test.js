import React from 'react';
import { shallow } from 'enzyme'
import LoginForm from './LoginForm';

describe('LoginForm', () => {
  it('should', () => {
    expect(true).toEqual(true)
  })
})