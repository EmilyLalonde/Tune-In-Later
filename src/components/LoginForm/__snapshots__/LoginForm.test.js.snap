// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginForm should match the snapshot 1`] = `
<form
  className="login-form"
>
  <input
    className="formInput"
    name="email"
    onChange={[Function]}
    placeholder="Please enter email address"
    type="text"
    value=""
  />
  <input
    className="formInput"
    name="password"
    onChange={[Function]}
    placeholder="Please enter password"
    type="text"
    value=""
  />
  <button
    className="login-button"
    onClick={[Function]}
  >
    SUBMIT
  </button>
</form>
`;
